### PR TITLE
build: use pnpm package manager

### PR DESCRIPTION
**Pull request by tisonkun**
This closes https://github.com/GreptimeTeam/docs/issues/720.

### CODE REVIEW

**Review by alili — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -49,11 +49,11 @@ This project requires Node.js version 18.0.0 or higher.
 Use the following commands to start a local server:
```
Comment (nicecui) [inline CONTRIBUTING.md:49]:
```suggestion
Use `npm install -g pnpm` to install package manager, and start a local server with the following commands:
```

**Comment by tisonkun** [comment]
Updated. CI failure fixed. Please retrigger the workflows.

**Comment by tisonkun** [comment]
CI passed. May you help merging this patch @nicecui?

**Review by nicecui — APPROVED** [review]

**Pull request merged**